tasks/banana.js

```diff
@@ -26,7 +26,8 @@ module.exports = function ( grunt ) {
 				sourceMessageKeys = Object.keys( sourceMessages ),
 				count = 0;
 
-			messageCount += sourceMessageKeys.length;
+			// Subtract one from the count for the @metadata key
+			messageCount += sourceMessageKeys.length - 1;
```

Comment:
Avoid relying on the subsequent code removing 1 key (and only one). Instead, move it down 10 lines to after `sourceMessageKeys.splice( , 1 );`. Then the natural length will be accurate.